src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('li>ul>li');
+const topHeaders = document.querySelectorAll('ul>li');
+
+function spanWrap(input) {
+  const text = input.innerHTML;
+  let newText = '<span>';
+  let count = 0;
+
+  for (const ch of text) {
```

Comment:
This for loop can be replaced with a more efficient string function, like 'replace'. It will make the code cleaner and more efficient.